events/templates/events/events.html

```diff
@@ -1,74 +1,103 @@
+            
+            <!-- Toggler/collapsible Button -->
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            
+            <!-- Navbar links -->
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Find Out Events</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">My events</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
 
-{% block title %}
-Event List
-{% endblock %}
+    <main>
+        {% block content %}
+        <!-- Default content goes here, if any -->
+        <div class="container">
+        <div class="sticky-note-container-big mt-5">
+            <h1>Events</h1>
+            <ul>
+                {% for event in events %}
+                <li class="event-item">
+                    <a href="{% url 'events:event-detail' event.id %}"><strong>{{ event.event_name }}</strong></a>
+                    <p>Event Location: {{ event.event_location }}</p>
+                    <p>Start time: {{ event.start_time }}</p>
+                    <p>End Time: {{ event.end_time }}</p>
+                    <p>Capacity: {{ event.capacity }}</p>
+                    {% if user.is_authenticated and user == event.creator %}
+                        <div class="button-container">
+                            <form method="post" action="{% url 'events:delete-event' event.id %}" onsubmit="return confirm('Are you sure you want to delete this event?');">
+                                {% csrf_token %}
+                                <input type="hidden" name="action" value="delete">
+                                <button class="btn btn-primary" type="submit">Delete Event</button>
+                            </form>
+                            <form method="GET" action="{% url 'events:update-event' event.id %}">
+                                {% csrf_token %}
+                                <input type="hidden" name="action" value="delete">
+                                <button class="btn btn-primary" type="submit">Update Event</button>
+                            </form>
+                        </div>
+                    {% endif %}
+                </li>
+                {% endfor %}
+            </ul>
+        </div>
+        <div id="map"></div>
```

Comment:
The map looks great!!!! Thank you! Just a minor thing, could you make the location of it as "fixed" so that it will always stay at the right of the page when we scroll down?

Reply:
Sure no problem!

Reply:
I think its fixed now